apps/speed.c

```diff
@@ -2841,15 +2889,15 @@ int speed_main(int argc, char **argv)
             ERR_print_errors(bio_err);
             op_count = 1;
         } else {
-            pkey_print_message("private", "rsa",
+            pkey_print_message("private", "rsa sign",
                                rsa_keys[testnum].bits, seconds.rsa);
             /* RSA_blinding_on(rsa_key[testnum],NULL); */
             Time_F(START);
             count = run_benchmark(async_jobs, RSA_sign_loop, loopargs);
             d = Time_F(STOP);
             BIO_printf(bio_err,
                        mr ? "+R1:%ld:%d:%.2f\n"
-                       : "%ld %u bits private RSA's in %.2fs\n",
+                       : "%ld %u bits private RSA sign's in %.2fs\n",
```

Comment:
I do not think the apostrophe in `RSA sign's`  is right. Similarly in the below cases.

Reply:
Hmm, I'm going to argue devil's advocate here:

An apostrophe is to stand in for missing letters (e.g. `it is` -> `its`) or for possessive (`Tom's bicycle`, although not in `his`, `hers`, `its`)

In this case, the full phrase would be `xxx bits private RSA signatures in ...` so the apostrophe _is_ standing in for missing letters!

But I agree, it reads horribly, and I would prefer something else here too

Reply:
So to what do you want me to change it? 
Just no apostrophe? Or spelled out `RSA signatures`, `RSA verifications`, `RSA encryptions`, `RSA decryptions` ?

Reply:
Well, TBH the previous `RSA's` isn't correct, although it is "less wrong" ? or at least seen enough that it could be considered "acceptable by some" to use an apostrophe for plural of an acronym like this.

For signatures, `N M-bit RSA signs` would match what is done for DSA, ECDSA, etc, and seems fine.

Those other algs use singular `verify` which makes me shudder when I read it!

Now, it's also common to abbreviate with a dot (period)  (most often with things such as `e.g.`) - so we could do something like

```
Doing 2048-bit private RSA sigs. for 10s: 17637 2048-bit RSA sigs. in 9.99s
Doing 2048-bit public RSA verifs. for 10s: 712226 2048-bit RSA verifs. in 9.99s
```

The other question of course is: could anyone be consuming this output by script in a CI, and would changes here break that? (Actually @zorrorffm do _we_ do that?)

Reply:
How about `xxx bits private RSA sign ops in ...` ? So the plural would be on `operations `(abbreviated as `ops`), not on sign, verify, etc.....

Reply:
> The other question of course is: could anyone be consuming this output by script in a CI, and would changes here break that? (Actually @zorrorffm do _we_ do that?)

Yes, our CI consumes the final output, the changes at line 4101 will break it. Thanks for the reminder, I will change our CI accordingly after this PR is merged.

Reply:
So I can easily change the code to use `xxx bits private RSA sign ops in ...` instead, but the question is: Only for RSA? That would be somewhat inconsistent with the other algos. Or should I change it for all algos to use `...ops` ? That would be a slightly larger change (and not really related to the topic of this PR). I guess I would then do this in a 2nd commit. 

Let me know what you think and if I should change it to `... ops` and where.

Reply:
@t8m @paulidale any opinion about how to continue with this?

Reply:
I am happy with `sign ops` and `verify ops` with no punctuation

Reply:
Let's go with @tom-cosgrove-arm's suggestion.

Reply:
OK, and just for the RSA places that I touch anyway, or for all algos ?

Reply:
I think if we're going to change output, let's be consistent, and let's do it in one go (if it breaks something, better for folks to just have to fixup once)

So I think yes, all algs

Reply:
> OK, and just for the RSA places that I touch anyway, or for all algos ?

I would leave that up to you